packages/amazonq/.changes/next-release/Bug Fix-80f16b7a-440d-4e06-89ad-1454c58ff28f.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
```

```suggestion
	"type": "Feature",
```

Thank you for contributing! This is not exactly a "bug" per say, it is by design